src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -4,14 +4,18 @@
 -->
 
 <template>
-	<div class="wrapper" :class="{'wrapper--big': isBig}">
+	<div class="wrapper"
+		:class="{'wrapper--big': isBig}"
+		@mouseover.stop="mouseover = true"
+		@mouseleave.stop="mouseover = false">
 		<TransitionWrapper name="fade">
 			<div v-if="showRaiseHandIndicator" class="status-indicator raiseHandIndicator">
 				<HandBackLeft :size="18" fill-color="#ffffff" />
 			</div>
 		</TransitionWrapper>
 
-		<div v-if="!isSidebar" class="bottom-bar">
```

Comment:
revert to one-line or bring back mouseover listeners here?
Still weird for me to have this 50px-high freeze line, but it's your call =)

Reply:
so you are suggesting to reduce the height here or what ? I am a bit confused 🙈

Reply:
Just revert a change on this line